src/node/rpc/frontend.h

```diff
@@ -392,6 +390,14 @@ namespace ccf
         {
           return;
         }
+        else
+        {
+          // Only register calls to existing endpoints
+          if (attempts == 1)
+          {
+            endpoints.increment_metrics_calls(*ctx);
+          }
+        }
```

Comment:
```suggestion
        // Only register calls to existing endpoints
        if (attempts == 1)
        {
          endpoints.increment_metrics_calls(*ctx);
        }
```

Reply:
@MahatiC I did this deliberately to make it clear it's the other branch where the endpoint isn't found. I think this is less obvious when flattened, and could be separated/re-ordered more easily in the future. So unless there is a strong objection, I would prefer to keep it that way.